js-classes/inheritance-es5.js

```diff
@@ -0,0 +1,71 @@
+function Car(name, model, year, color, maxSpeed, fuelCapacity, fuelConsumption){
+	this.name = name;
+	this.model = model;
+	this.year = year;
+	this.color = color;
+	this.maxSpeed = maxSpeed;
+	this.fuelCapacity = fuelCapacity || 60;
+	this.fuelConsumption = fuelConsumption || 10;
+}
+
+Car.prototype.getFullName = function(name, model){
+	console.log(this.model + ' ' + this.name);
+}
+
+Car.prototype.getAge = function(){
+var date = new Date;
+var year = date.getFullYear();
+return year - this.year
+}
+
+Car.prototype.changeColor = function(color){
+	if (this.color == 'red') {
```

Comment:
You should compare an existing color `this.color` with the new chosen `color` (from the function parameter)